README.md

```diff
@@ -97,16 +97,18 @@ $ portmap &
 $ cdmiservice &
 
 $ /usr/bin/chromium/chrome --no-sandbox  \
-    --use-gl=egl --ozone-platform=wayland --no-sandbox --composite-to-mailbox --in-process-gpu --enable-low-end-device-mode \
+    --use-gl=egl --ozone-platform=wayland --no-sandbox --composite-to-mailbox \
+    --in-process-gpu --enable-low-end-device-mode \
     --enable-logging --v=0 \
     --start-maximized \
     --user-data-dir=data_dir \
     --blink-platform-log-channels=Media\
-    --register-pepper-plugins="/usr/lib/chromium/libopencdmadapter.so#ClearKey CDM#ClearKey CDM0.1.0.0#0.1.0.0;application/x-ppapi-open-cdm" \
+    --register-pepper-plugins="/usr/lib/chromium/libopencdmadapter.so#ClearKey \
```

Comment:
Have you tested this command? It adds additional spaces to the input string passed to the register-pepper-plugins argument.